.circleci/config.yml

```diff
@@ -146,7 +146,8 @@ jobs:
       - run: make test-runtime-image gateway-logs --keep-going
       - login-docker:
           command: |
-            IMAGE_TAG="${CIRCLE_TAG:-${CIRCLE_BRANCH}}"
+            CIRCLE_BRANCH_LOWER=`echo $CIRCLE_BRANCH | tr '[:upper:]' '[:lower:]'`
```

Comment:
I do not think that these changes should be part of this PR, maybe a separate commit or separate PR. Why is needed?

Reply:
The s2i-runtime failed because it used the branch name as a tag for the docker image, but it fails with uppercase characters. This change is to transform them to lowercase

Reply:
:+1:

Reply:
I separated the circleci changes on a different commit